src/core/thread/key_manager.hpp

```diff
@@ -485,6 +570,11 @@ class KeyManager : public InstanceLocator, private NonCopyable
         kOneHourIntervalInMsec     = 3600u * 1000u,
     };
 
+#if OPENTHREAD_CONFIG_PSA_CRYPTO_ENABLE
+    enum {kMasterKeyPsaItsOffset = OPENTHREAD_CONFIG_PSA_ITS_NVM_OFFSET + 1,
```

Comment:
`OPENTHREAD_CONFIG_PSA_ITS_NVM_OFFSET` is not defined.

Reply:
NVM offset is vendor specific and the idea is to let the vendors decide what it needs to be. But thinking back, it is good to have a default offset defined I guess. I will add it to the PR.